### PR TITLE
Wb fix missing alttext mmd

### PR DESCRIPTION
**Pull request by MeeraDi**

### CODE REVIEW

**Comment by MeeraDi** [comment]
Hi, @Rick-Anderson, @mcleblanc, @v-andreaco
This PR is ready for final review.

File: iis/extensions/configuring-application-request-routing-arr/achieving-high-availability-and-scalability-arr-and-nlb.md [hunk 1]
```diff
@@ -105,40 +105,40 @@ The NLB configuration is divided into the following steps:
...
-    [![](achieving-high-availability-and-scalability-arr-and-nlb/_static/image33.jpg)](achieving-high-availability-and-scalability-arr-and-nlb/_static/image32.jpg)
+    ![Screenshot of the Port Rules dialog](achieving-high-availability-and-scalability-arr-and-nlb/_static/image32.jpg)
 11. Now that the NLB cluster is created, you can add additional members to the cluster. Follow the remaining steps on all additional member servers. In the Network Load Balancing Manager, right-click on the newly selected cluster, and then select **Add Host To Cluster**.  
-    [![](achieving-high-availability-and-scalability-arr-and-nlb/_static/image35.jpg)](achieving-high-availability-and-scalability-arr-and-nlb/_static/image34.jpg)
+    ![Screenshot of the Add Host to Cluster dialog.](achieving-high-availability-and-scalability-arr-and-nlb/_static/image34.jpg)
 12. Type the server address of the member to be added. If there are multiple interfaces, select the one that should be used by the NLB cluster.  
     [![](achieving-high-availability-and-scalability-arr-and-nlb/_static/image37.jpg)](achieving-high-availability-and-scalability-arr-and-nlb/_static/image36.jpg)
```
Comment (Rick-Anderson) [inline iis/extensions/configuring-application-request-routing-arr/achieving-high-availability-and-scalability-arr-and-nlb.md:143]: missing, please review the build report

Comment (Rick-Anderson) [reply iis/extensions/configuring-application-request-routing-arr/achieving-high-availability-and-scalability-arr-and-nlb.md:143]: For this file
image

File: iis/develop/windows-web-application-gallery/introducing-the-windows-web-application-gallery.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ The Web Application Gallery integrates with the [Web Platform Installer 3.0](htt
 
 The Web Application Gallery also provides a way for developers to reach millions of Windows users. When an application is accepted by the Web Application Gallery, the application is added to the Web Application Gallery ATOM feed. The ATOM feed is consumed by the Web Application Gallery itself, Web Platform Installer 3.0, WebMatrix, IIS Manager, and participating Hosting Control Panels.
```
Comment (Rick-Anderson) [inline iis/develop/windows-web-application-gallery/introducing-the-windows-web-application-gallery.md:21]: For this file, from the build report
Line 4: [Suggestion-[PR only](https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/pr-only?branch=main)] Attribute 'description' with value 'The Windows Web Application Gallery makes it easy to explore, discover and install popular community ASP.Net and PHP applications on Windows. Users can brows...' is duplicated in 'develop/windows-web-application-gallery/introducing-the-windows-web-application-gallery.md(4,14)', 'develop/windows-web-application-gallery/windows-web-application-gallery-principles.md(4,14)'.

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

File: iis/extensions/configuring-application-request-routing-arr/achieving-high-availability-and-scalability-arr-and-nlb.md (hunk 1 above)
Comment (MeeraDi) [reply iis/extensions/configuring-application-request-routing-arr/achieving-high-availability-and-scalability-arr-and-nlb.md:143]: @Rick-Anderson the rest of this file will be addressed in another PR.

**Comment by MeeraDi** [comment]
@Rick-Anderson 
cc: @v-andreaco 
I updated the description metadata. The other missing alttext will be fixed in another PR.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**